make/github.mk

```
@@ -12,8 +12,8 @@ GET_GITHUB_RELEASE_FN = get_github_release() { \
 		for i in $$(seq $$attempts); do \
 			curl --silent --show-error --fail --location --output $${1} $${2} && break ;\
 			[ $$i -eq $$attempts ] && exit 1; \
```

Thinking of consistent bash style.

I'd suggest to always use `[[` instead of `[` in bash because the former is built-in while the latter starts a command.
I'd also suggest to quoting any variables.

```suggestion
			[[ "$$i" -eq "$$attempts" ]] && exit 1; \
```

---

Thanks. I changed to `[[` here and improved the quoting, but `[[` differs from `[` w.r.t. word splitting, so I didn't add them in this particular place.